src/types/entities/agents/manager.ts

```diff
@@ -85,6 +85,12 @@ export interface AgentManagerOptions {
      */
     distinctId?: string;
     mode?: ChatMode;
+    /**
+     * Desired stream resolution for the session
+     * @minimum 0
+     * @maximum 1080
+     */
+    streamResolution?: number;
```

Comment:
Can we support all resolutions?
do we want to?
maybe limit to
144 | 360 | 720 | 1080
like youtube?

Reply:
there is a limit of minimum resolution but we don't acutally limit it.
maybe let's keep it as number between 0 and 1080 cause the api supports all and then think of limit it to what you were sasying.
is that ok?  @HaMMeRSI